components/dashboard/donut.tsx

```diff
@@ -0,0 +1,86 @@
+import { Chart as ChartJS, ArcElement, Legend, Tooltip } from "chart.js";
+import { Doughnut } from "react-chartjs-2";
+import { defi } from "../../utils/tzktHooks";
+
+ChartJS.register(ArcElement, Tooltip, Legend);
+
+type DonutProperty = {
```

Comment:
Idem pour Props

Reply:
fixed in:
https://github.com/marigold-dev/tzsafe-ui/pull/167/commits/b25f92837edad6ec30404151f46ae961dc208fe8